frontend/src/components/common/footer/index.css

```diff
@@ -82,27 +82,53 @@
   text-align: center;
 }
 
-@media (min-width: 768px) and (max-width: 2560px) {
+@media (min-width: 769px) and (max-width: 2560px) {
   .tsocial {
     display: none;
   }
 }
 
-@media only screen and (max-width: 520px) {
+@media only screen and (max-width: 768px) {
+  .footer-list {
+    font-size: 12pt;
+  }
   .footer-link {
     display: inline-block;
     margin: 2vh;
   }
-  .svg-inline--fa.fa-facebook-square.fa-w-14.facebook,
+  .svg-inline--fa.fa-facebook.fa-w-16.facebook,
   .svg-inline--fa.fa-twitter.fa-w-16.twitter,
   .svg-inline--fa.fa-instagram.fa-w-14.insta {
-    width: 8vh;
-    height: 8vh;
+    width: 30px;
+    height: 30px;
     background: #fff;
     border-radius: 50%;
-    padding: 1.5vh;
     margin: 1vh;
   }
+  .svg-inline--fa.fa-facebook.fa-w-16.facebook {
+    color: #3c579d;
+    background-color: #fff;
+    border: 0.5px solid #fff;
+  }
+  .svg-inline--fa.fa-twitter.fa-w-16.twitter {
+    background-color: #52a7e7;
+    color: #fff;
+    padding: 1vh;
```

Comment:
can we use PX instead of `vh` here. Its more of a common standard to pretty much always use px except for maybe heights. 

I think `3px` looks solid here, but feel free to go with what you think is best